lib/stripe/resources/account.rb

```diff
@@ -10,7 +10,6 @@ module Stripe
   # account has started to go through Connect Onboarding. Once you create an [Account Link](https://stripe.com/docs/api/account_links) or [Account Session](https://stripe.com/docs/api/account_sessions),
   # some properties are only returned for Custom accounts. Learn about the differences [between accounts](https://stripe.com/docs/connect/accounts).
   class Account < APIResource
-    extend Gem::Deprecate
```

Comment:
Weird, how did this unused extend end up here in the first place?

Reply:
It was added as an override (I assume for a method that no longer exists since it was deprecated) -- I removed it in my PR when going through overrides.